### ISSUE TITLE
problem with @interface

### ISSUE THREAD

**Issue by Tedko** [drkameleon/iswift#61]
```objective-c
@interface Dot()
{
    CGFloat x;
}
@end
```

will trigger an error

**Comment by drkameleon** [comment]
I'm looking into it. What type of error did it throw?

**Comment by drkameleon** [comment]
Issue fixed as of the upcoming 1.8, provide that the interface type has already been declared as a type (e.g. via `#import`ing the relevant header `#import "Dot.h"` or by adding the interface itself, like `@interface Dot ....`. 

----

**Example:**

```objective-c
@interface Dot
@end

@interface Dot()
{
    CGFloat x;
}
@end

@implementation Dot
@end
```

**Output:**

```swift
class Dot {

	var x: CGFloat?
	
}
```
